src/hooks/useFollowScroll.ts

```
@@ -0,0 +1,62 @@
+import { useRef, useEffect, useState } from "react";
+import useDeviceSize from "./useDeviceSize";
+
+export default function useFollowScroll<T extends HTMLElement>(startFollowPosition: number = 200) {
```

Biome

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)